modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2302,6 +2302,18 @@ export class Wallet implements IWallet {
       this.bitgo.setRequestTracer(params.reqId);
     }
 
+    // Split query if we can't enable multiple tokens in one tx
+    if (!teConfig.supportsMultipleTokenEnablements && params.enableTokens.length > 1) {
+      const queries = params.enableTokens.map(async (enableToken) => {
+        return this.buildTokenEnablements({
+          ...params,
+          enableTokens: [enableToken],
+        });
+      });
+      const results = await Promise.all(queries);
+      return results.flat();
```

Comment:
Is it intended that these functions all end the parent function and don't hit `delete buildParams.enableTokens;`